### PR TITLE
Doc v2 g

### PR DESCRIPTION
**Pull request by jtlap**
by error contains f also

### CODE REVIEW

File: include/eve/module/core/regular/fdim.hpp [hunk 1]
```diff
@@ -43,34 +43,33 @@ namespace eve
...
-//!     * `x`, `y` :  [real](@ref eve::value)  arguments.
+//!     * `x`, `y`: [real](@ref eve::value)  arguments.
+//!     * `c`: [Conditional expression](@ref conditional_expr) masking the operation.
+//!     * `m`: [Logical value](@ref logical) masking the operation.
 //!
 //!   **Return value**
 //!
-//!     Returns the  [elementwise](@ref glossary_elementwise) positive difference
-//!     between `x` and `y`:
+//!     Returns the positive difference between `x` and `y`:
```
Comment (jfalcou) [inline include/eve/module/core/regular/fdim.hpp:63]: 1. and 2. ??

File: include/eve/module/core/regular/firstbitunset.hpp [hunk 2]
```diff
@@ -40,33 +40,30 @@ namespace eve
...
 //!
-//!    **Return value**
+//!   **Return value**
 //!
-//!       Computes [elementwise](@ref glossary_elementwise) the bit pattern in
-//!       which the only bit set (if it exists) is
-//!       the first bit unset (beginning with the least significant bit) in the input.
+//!      1. Computes [elementwise](@ref glossary_elementwise) the bit pattern in
+//!         which the only bit set (if it exists) is
+//!         the first bit unset (beginning with the least significant bit) in the input.
```
Comment (jfalcou) [inline include/eve/module/core/regular/firstbitunset.hpp:62]: why the carriage return ??

File: include/eve/module/core/regular/fma.hpp [hunk 3]
```diff
@@ -13,31 +13,28 @@
...
-{
-  template<eve::value T,eve::value U,eve::value V>
-  requires(Options::contains(promote))
-  constexpr EVE_FORCEINLINE auto operator()(T a, U b, V c) const noexcept { return EVE_DISPATCH_CALL(a,b,c); }
+  template<typename Options>
+  struct fma_t : strict_elementwise_callable<fma_t, Options, pedantic_option, promote_option>
+  {
+    template<eve::value T,eve::value U,eve::value V>
+    requires(Options::contains(promote))
+      constexpr EVE_FORCEINLINE auto operator()(T a, U b, V c) const noexcept { return EVE_DISPATCH_CALL(a,b,c); }
```
Comment (jfalcou) [inline include/eve/module/core/regular/fma.hpp:21]: why the inconsistent tabulation ? Don't change code for no reason.

File: include/eve/module/core/regular/fms.hpp [hunk 4]
```diff
@@ -28,71 +28,53 @@ namespace eve
...
+//!         has no hardware capability.
+//!      4. TO DO : DESCRIBE
+//!
+//!  @groupheader{External references}
+//!   *  [C++ standard reference](https://en.cppreference.com/w/cpp/numeric/special_functions/fma)
+//!
+//!  @groupheader{Example}
+//!  @godbolt{doc/core/fma.cpp}
+//! @}
+  inline constexpr auto fms = functor<fms_t>;
```
Comment (jfalcou) [inline include/eve/module/core/regular/fms.hpp:77]: missing //=========================

File: include/eve/module/core/regular/fnms.hpp [hunk 5]
```diff
@@ -29,16 +29,13 @@ namespace eve
     EVE_CALLABLE_OBJECT(fnms_t, fnms_);
   };
 
-//================================================================================================
-//! @addtogroup core_fma_family
+//======================================================================================================================
+//! @addtogroup core_fnms_family
```
Comment (jfalcou) [inline include/eve/module/core/regular/fnms.hpp:33]: wrong group

File: include/eve/module/core/regular/frac.hpp [hunk 6]
```diff
@@ -44,51 +44,42 @@ namespace eve
...
-//!   * Masked Call
-//!
-//!     The call `eve::frac[mask](x)` provides a masked version of `eve::frac` which is
-//!     equivalent to `if_else (mask, frac(x), x)`.
+//!     1. The fractional part of `x` with the same sign as `x`: `x-trunc(x)`. In particular:
+//!        * If an [element](@ref glossary_elementwise) of `x` is \f$\pm0\f$,  \f$\pm0\f$ is
+//!          returned.
+//!        * If an [element](@ref glossary_elementwise) of `x` is \f$\pm\infty\f$ or `Nan`, a
+//!          `Nan` is returned.
+//!     2. [The operation is performed conditionnaly](@ref conditional).
```
Comment (jfalcou) [inline include/eve/module/core/regular/frac.hpp:76]: Where are 3. and 4. ?

File: include/eve/module/core/regular/frexp.hpp [hunk 7]
```diff
@@ -41,35 +40,30 @@ struct frexp_t : elementwise_callable<frexp_t, Options, pedantic_option, raw_opt
...
 //!
-//!     * `x` :  [argument](@ref eve::value).
+//!     * `x` :  [floating value](@ref floating_value).
 //!
 //!   **Return value**
 //!
-//!     Computes the [elementwise](@ref glossary_elementwise) ieee frexp of the floating value,
-//!     returning a pair `{m,e}`  of values ` of the same type as `x`, which are related by
-//!     \f$x =  m\times 2^e\f$, with  \f$|m| \in [0.5, 1.5[\f$.
+//!     1. returns a pair `{m,e}`  of values ` of the same type as `x`, which are related by
```
Comment (jfalcou) [inline include/eve/module/core/regular/frexp.hpp:54]: a kumi::tuple or else ppl will try to assign it to std::pair

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: include/eve/module/core/regular/fdim.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/core/regular/fdim.hpp:63]: done

File: include/eve/module/core/regular/firstbitunset.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/module/core/regular/firstbitunset.hpp:62]: done

File: include/eve/module/core/regular/fma.hpp (hunk 3 above)
Comment (jtlap) [reply include/eve/module/core/regular/fma.hpp:21]: done

File: include/eve/module/core/regular/fms.hpp (hunk 4 above)
Comment (jtlap) [reply include/eve/module/core/regular/fms.hpp:77]: done

File: include/eve/module/core/regular/frac.hpp (hunk 6 above)
Comment (jtlap) [reply include/eve/module/core/regular/frac.hpp:76]: done

File: include/eve/module/core/regular/frexp.hpp (hunk 7 above)
Comment (jtlap) [reply include/eve/module/core/regular/frexp.hpp:54]: done

File: include/eve/module/core/regular/fnms.hpp (hunk 5 above)
Comment (jtlap) [reply include/eve/module/core/regular/fnms.hpp:33]: done

**Review by jfalcou — APPROVED** [review]

File: include/eve/module/core/regular/fnms.hpp (hunk 5 above)
Comment (jfalcou) [reply include/eve/module/core/regular/fnms.hpp]: the group is core_fma_family not core_fnms_family. This is still to be fixed

**Review by jfalcou — APPROVED** [review]

**Pull request merged**